beta2-quickstart-master/frontend/src/App.tsx

```diff
@@ -9,58 +9,102 @@ import { CounterContractAbi__factory } from "./contracts";
 const CONTRACT_ID =
   "0x3edb96c23766b8504caaff042994efa18460e7ba27f60191394a6bcf5be8d7d8";
 
-//the private key from createWallet.js
-const WALLET_SECRET =
-  "0xc4a69e0cc4ce1e0b45d25899b3cedced332d193c8a5c706187ffd50aa7591ce6";
-
-// Create a Wallet from given secretKey in this case
-// The one we configured at the chainConfig.json
-const wallet = Wallet.fromPrivateKey(
-  WALLET_SECRET,
-  "https://node-beta-2.fuel.network/graphql"
-);
-
-// Connects out Contract instance to the deployed contract
-// address using the given wallet.
-const contract = CounterContractAbi__factory.connect(CONTRACT_ID, wallet);
+declare global {
+  interface Window{
+    fuel?:any
+  }
+}
```

Comment:
I think we can remove this and add it into the install step `@fuels-wallet/sdk` and then import it directly on this file if we don't want to add the step to add it on tsconfig on the sample. This will avoid using any type.

Reply:
can you elaborate here @luizstacio with a little more details / context? Thanks.

Reply:
@dabit3 If I'm understanding correctly, Luiz is pointing towards installing the `@fuels-wallet/sdk` for type completions and avoiding `any`.  Ref: https://wallet.fuel.network/docs/typescript/

Reply:
Hey, yes thanks, this should have been resolved in my update